src/jit/codegenarmarch.cpp

```diff
@@ -1009,7 +1017,7 @@ void CodeGen::genPutArgSplit(GenTreePutArgSplit* treeNode)
         nextIndex += 1;
     }
 
-    // Set registers
+    // We set registers to assign the last target register at the last instruction.
```

Comment:
I would suggest rewording this to:
```
// We set up the registers in order, so that we assign the last target register after `baseReg` is no longer in use,
// in case we had to reuse the last target register for it.
```
It's a bit verbose, but this is a bit confusing, so it's better to err on the side of too much description, I think.